test/modules/layers/test_quantization.py

```diff
@@ -45,7 +44,7 @@ def setUp(self):
         )
 
     def test_quantized_output(self):
-        self.vq.embedding = nn.Embedding.from_pretrained(self.embedding_weights)
+        self.vq.embedding = self.embedding_weights
```

Comment:
Since you are reusing `self.vq.embedding` for the subsequent tests, should we just move this into `setUp` in case one day we remove this assignment and the other tests will break?

Reply:
I only need pre-initialized weights for this test, the others depend on init_embedding and would ignore this anyway